deploy/packages/compilers.yaml

```diff
@@ -52,4 +52,4 @@ packages:
       - architecture
       - compiler
     specs:
-      - llvm@9.0.0
+      - 'llvm@11.0.0 build_type=RelWithDebInfo +cuda cuda_arch=70 +omp_tsan +python ^cuda@10.1.243'
```

Comment:
We're building a lot of stuff in compilers, then again in externals. I'm thinking that non-essential compilers could actually be moved to externals (next deployment)

CUDA and Python come to mind...

Reply:
thats true! May be in the next deployment we can adjust this?

By the way, llvm compiles fine but [tools stage fails](https://bbpcode.epfl.ch/ci/blue/organizations/jenkins/hpc.spack-deployment.pull-requests/detail/hpc.spack-deployment.pull-requests/1207/pipeline) with:

```

[2020-11-07T13:51:59.664Z] 5jk2xjg zlib@1.2.11+optimize+pic+shared
[2020-11-07T13:51:59.664Z] ### 14:51:59 gathering specs
[2020-11-07T13:52:21.785Z] ==> Error: No compilers with spec gcc@8.3.0 found for operating system rhel7 and target x86_64.
[2020-11-07T13:52:21.785Z] Run 'spack compiler find' to add compilers.
script returned exit code 1
```

If I re-trigger or start tools from scratch (by removing dir) then I believe it will finish OK. But I kept the state as it is in case that helps to see what is possible issue.

Reply:
Hm, looks weird. I'd just remove + retrigger...

Reply:
you can see it's reproducible. If I remove now only tools directory and re-launch then it should work fine.

Reply:
FWIW, I just noticed that in my merge branch, `ispc` also depends on `llvm@11:`. That means we build a bunch of Pythons in `compilers`, just for LLVM. Then in `externals`, we re-build Python and LLVM :facepalm: .